src/server/svr_mail.c

```diff
@@ -636,10 +636,10 @@ svr_mailowner_id(char *jid, job *pjob, int mailpoint, int force, char *text)
 	 * From here on, we are a child process of the server.
 	 * Fix up file descriptors and signal handlers.
 	 */
-
+	net_close(-1);
 	if(pfn_rpp_terminate)
 		rpp_terminate();
-	net_close(-1);
+
```

Comment:
I'm confused. You had previously placed the rpp_terminate() calls AFTER the call to net_close(-1), but here you do the opposite. Why?

Reply:
Here too, the rpp_terminate() is after the call to net_close(), no?

Reply:
You're right. Please add a space between the if and the open paren.

Reply:
Thanks. Done!